maps/predship/regular.dmm

```diff
@@ -1177,16 +1177,16 @@
 /area/yautja)
 "dp" = (
 /obj/structure/surface/table/reinforced,
-/obj/item/XenoBio/Blood,
-/obj/item/XenoBio/Blood,
+/obj/item/xeno_piece/blood,
+/obj/item/xeno_piece/blood,
 /turf/open/floor/holofloor{
 	dir = 2;
 	icon_state = "cult"
 	},
 /area/yautja)
 "dq" = (
 /obj/structure/surface/table/reinforced,
-/obj/item/XenoItem,
+/obj/item/xeno_item,
```

Comment:
The desc makes it sound like an abstract one (prompting opening a bug report) - i think it's worth either making it so and changing this one to another, or clearing that up from the description